opennms-base-assembly/src/main/filtered/etc/vacuumd-configuration.xml

```diff
@@ -416,7 +428,16 @@
            AND statustime &lt; now() - interval '24 hours'
       </statement>
     </action>
-    
+
+    <action name="clearPathOutages" >
+      <statement>
+        UPDATE alarms
+           SET severity = 2, lastautomationtime = ${_ts}
+         WHERE eventuei = 'uei.opennms.org/nodes/pathOutage'
```

Comment:
what about tweak the trigger to have the following ?

AND GREATEST(lastautomationtime, lasteventtime) &lt; now() - interval '1 minute'